### DISCUSSION TITLE
IMPORT node install with Analyst Bench leaves Steno, Suricata and Zeek DISABLED

### DISCUSSION THREAD

**Discussion by vanh20** [Security-Onion-Solutions/securityonion#7753] [Unsupported Versions] [answered]
Sorry guys. I don't know why I totally [missed](https://docs.securityonion.net/en/2.3/so-status.html?highlight=IMPORT%20node) it.

Update:
Yes, I think I do know now since I'm not the only [one](https://github.com/Security-Onion-Solutions/securityonion/discussions/4204) and I was hoping to find that information [here](https://docs.securityonion.net/en/2.3/so-import-pcap.html). Would it be possible to make mention of this IMPORT Node [here](https://docs.securityonion.net/en/2.3/so-import-pcap.html) where I think it is more related to since people tend to think of the task they need to accomplish more than the result of just any diagnostic test? Just for documentation clarity.

Big thanks to the developers.

**Comment by dougburks** [comment, marked as answer]
I've updated https://docs.securityonion.net/en/2.3/so-import-pcap.html#configuration to address this.

**Reply by vanh20** [reply]
Thanks Doug! Security Onion keeps getting better and better! :1st_place_medal: :)
